### PR TITLE
#2480 PDF viewer selection text color 

### PR DESCRIPTION
**Pull request by ppatel221**
@humphd I've created a new PR.

### CODE REVIEW

**Comment by flukeout** [comment]
Testing this now.

**Comment by flukeout** [comment]
Hey @ppatel221 - thanks for submitting! I've left a few comments on your code, if you make the changes, then we are good to go!

File: src/extensions/extra/PDFView/styles/viewer-overrides.css [hunk 1]
```diff
@@ -2,6 +2,16 @@
  * Various overrides of the built viewer.css file. Keep this as a CSS file vs. LESS
  */
 
+
+#pdf-meta *::selection  {
```
Comment (flukeout) [inline src/extensions/extra/PDFView/styles/viewer-overrides.css]: Please remove the extra space before the opening `{`

File: src/extensions/extra/PDFView/styles/viewer-overrides.css [hunk 2]
```diff
@@ -2,6 +2,16 @@
...
+    background: #abdffa !important;
```
Comment (flukeout) [inline src/extensions/extra/PDFView/styles/viewer-overrides.css]: In testing this, it seems that the rule takes effect without needing the `!important` declaration. Can you confirm on your end and remove `!important` if that is the case?

File: src/extensions/extra/PDFView/styles/viewer-overrides.css [hunk 3]
```diff
@@ -2,6 +2,16 @@
...
+}
+
+#pdf-meta *::-moz-selection {
+    background: #abdffa !important;
+
```
Comment (flukeout) [inline src/extensions/extra/PDFView/styles/viewer-overrides.css]: Please remove this extra blank line.

File: src/extensions/extra/PDFView/styles/viewer-overrides.css (hunk 2 above)
Comment (ppatel221) [reply src/extensions/extra/PDFView/styles/viewer-overrides.css]: It seems to be working fine without the !important declaration.

**Comment by ppatel221** [comment]
I've made the changes! Requested PR.

**Comment by humphd** [comment]
Re-flagging @flukeout for a followup review.

**Pull request merged**